tracer/build/_build/docker/gitlab/install_java.ps1

```diff
@@ -23,14 +21,13 @@ mkdir c:\tmp\java
 
 Write-Host -ForegroundColor Green "Extracting $out to c:\tmp\java"
 
-# Start-Process "7z" -ArgumentList 'x -oc:\tmp\java java.zip' -Wait
-Expand-Archive -Path $out -DestinationPath C:\tmp\java
+Start-Process "7z" -ArgumentList 'x -oc:\tmp\java java.zip' -Wait
```

Comment:
I'm _fairly_ confident that 7z isn't installed, hence using ExpandArchive here previously 🙂

Reply:
Pushed a fix for this: https://github.com/DataDog/dd-trace-dotnet/pull/4496/commits/58940078a06be8f75b1588119340be990c6c553b